byzcoin/streaming.go

```diff
@@ -129,7 +129,6 @@ func (s *Service) PaginateBlocks(msg *PaginateRequest) (chan *PaginateResponse,
 	stopChan := make(chan bool)
 
 	go func() {
```

Comment:
Shouldn't there be a `defer close(outChan)` in this go-routine? So the `return`s would inform the caller that something went wrong.

Hmm - in the other PR I supposed that the `outChan` never closes... What is the good thing to do here?

Reply:
It is the user's responsibility to check for any errors in a paginate response, and close the connection or send a new message. The connection is closed after 5 minutes of inactivity anyway.

Reply:
We should probably better define what the interface of the streaming service should actually be...